packages/flutter/lib/src/widgets/scroll_notification.dart

```diff
@@ -282,3 +282,13 @@ class UserScrollNotification extends ScrollNotification {
     description.add('direction: $direction');
   }
 }
+
+/// A predicate for [ScrollNotification], used to customize widgets that
+/// listen to notifications from their children.
+typedef bool ScrollNotificationPredicate(ScrollNotification notification);
+
+/// A [ScrollNotificationPredicate] that checks whether notification.depth == 0,
+/// which means that the notification diid not bubble through any intervening
+/// scrolling widgets.
+bool defaultScrollNotificationPredicate(ScrollNotification notification) =>
```

Comment:
nit: avoid => if it involves line breaks

Reply:
Done.